jobs/kola-aws.Jenkinsfile

```
@@ -0,0 +1,62 @@
+@Library('github.com/coreos/coreos-ci-lib') _
+
+def streams
+node {
+    checkout scm
+    streams = load("streams.groovy")
+    pod = readFile(file: "manifests/pod.yaml")
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             // list devel first so that it's the default choice
+             choices: (streams.development + streams.production + streams.mechanical),
+             description: 'Fedora CoreOS stream to test'),
+      string(name: 'VERSION',
+             description: 'Fedora CoreOS Build ID to test',
+             defaultValue: '',
+             trim: true),
+      string(name: 'S3_STREAM_DIR',
+             description: 'Fedora CoreOS S3 Stream Directory',
+             defaultValue: '',
+             trim: true),
+      string(name: 'COREOS_ASSEMBLER_IMAGE',
+             description: 'Override coreos-assembler image to use',
+             defaultValue: "coreos-assembler:master",
+             trim: true)
+    ])
+])
+
+currentBuild.description = "[${params.STREAM}] - ${params.VERSION}"
+
+def s3_stream_dir = params.S3_STREAM_DIR
+if (s3_stream_dir == "") {
+    s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
+}
+
+cosaPod(image: params.COREOS_ASSEMBLER_IMAGE, memory: "256Mi",
+        secrets: ["aws-fcos-builds-bot-config", "aws-fcos-kola-bot-config"]) {
+
+    def ami, ami_region
+    stage('Fetch Metadata') {
+        shwrap("""
+        export AWS_CONFIG_FILE=/run/kubernetes/secrets/aws-fcos-builds-bot-config/config
```

```suggestion
        export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
```

Is what we had in the past. Shall we keep it the same in case something changes we won't need to update this code, just the variable.

---

Same suggestion in GCP file